spec/services/import_polygons_spec.rb

```diff
@@ -1,85 +1,127 @@
+        end
       end
 
-      it "imports Tower Hamlets boundaries" do
-        expect(LocationPolygon.london_boroughs.first.boundary).to eq(points)
+      context "when location type is cities" do
+        let(:location_type) { :cities }
+        let(:boundary_points) { [51.406361958644, -2.3780576677997, 51.4063596372237, -2.3787764623145] }
+        let(:file_name) { "ons_cities.json" }
+
+        it "imports Bath city" do
+          expect(LocationPolygon.cities.first.name).to eq("bath")
+        end
+
+        it "does not import non-existent city" do
+          expect(LocationPolygon.cities.count).to eq(1)
+        end
+
+        it "imports Bath boundaries" do
+          expect(LocationPolygon.cities.first.boundary).to eq(boundary_points)
+        end
       end
     end
 
-    context "when location type is cities" do
-      let(:location_type) { :cities }
-      let(:points) { [51.406361958644, -2.3780576677997, 51.4063596372237, -2.3787764623145] }
-      let(:file_name) { "ons_cities.json" }
-
-      it "imports Bath city" do
-        expect(LocationPolygon.cities.first.name).to eq("bath")
+    context "when not stubbing #get_buffers" do
+      before do
+        distances = [5, 10, 15, 20, 25]
+        distances.each do |distance|
+          allow(HTTParty).to receive(:get).with(
+            "https://ons-inspire.esriuk.com/arcgis/rest/services/Utilities/Geometry/GeometryServer/buffer?"\
+             "bufferSR=3857&distances=#{DistanceHelper.convert_miles_to_metres(distance)}&f=json&geodesic=f"\
+             "alse&geometries=%7B%22geometryType%22%3D%3E%22esriGeometryPolygon%22%2C+%22geometries%22%3D%3E"\
+             "%5B%7B%22rings%22%3D%3E%5B%5B%5B55.8110853660943%2C+-2.0343575091738%5D%2C+%5B55.7647624900862%"\
+             "2C+-1.9841097397706%5D%5D%5D%7D%5D%7D&inSR=4326&outSR=4326&unionResults=true&unit=",
+          ).and_return(buffer_response)
+        end
+        subject.call
       end
 
-      it "does not import non-existent city" do
-        expect(LocationPolygon.cities.count).to eq(1)
+      let(:location_type) { :regions }
+      let(:boundary_points) { [55.8110853660943, -2.0343575091738, 55.7647624900862, -1.9841097397706] }
+      let(:file_name) { "ons_regions.json" }
+      let(:buffer_response) { JSON.parse(file_fixture("buffer_response.json").read) }
+      let(:buffer_coordinates) { [65.06414131400004, -12.5761721229999353, 65.06185937400005, -12.5761901179999427] }
+      let(:imported_buffers) do
+        {
+          "5" => buffer_coordinates,
+          "10" => buffer_coordinates,
+          "15" => buffer_coordinates,
+          "20" => buffer_coordinates,
+          "25" => buffer_coordinates,
+        }
       end
 
-      it "imports Bath boundaries" do
-        expect(LocationPolygon.cities.first.boundary).to eq(points)
+      it "imports buffers" do
+        expect(LocationPolygon.regions.first.buffers).to eq(imported_buffers)
```

Comment:
Can you change `LocationPolygon.regions.first` to `LocationPolygon.with_name('bath')`? The `first` threw me off when I was trying to read it - I thought this was only testing the first buffer.